packages/react/src/lib/getScrollParent.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Returns the parentNode or the host of the element
+ * @argument {Node} node
+ * @returns {Node} parent
+ */
+const getParentNode = (node: Node): Node => {
+  if (node.nodeName === 'HTML') {
+    return node
+  }
+
+  return node.parentNode || (node as any).host
+}
+
+/**
+ * Get CSS computed property of the given element
+ * @argument {Node} node
+ * @argument {string} property
```

Comment:
lets add short comments for the arguments, what they stand for.

Reply:
done, thanks